.github/workflows/windows-build-and-test.yaml

```diff
@@ -0,0 +1,57 @@
+# GitHub Actions configuration for TimescaleDB
+name: Build on Windows
+on:
+  push:
+    branches:
+      - prerelease_test
+  pull_request:
+jobs:
+  build:
+    name: PG${{ matrix.pg }} [${{ matrix.build_type }}]
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        pg: [ 9, 10, 11, 12 ]
```

Comment:
The different versions of 9 are separated in the documentation, and our tests. Maybe you should pick just `9.6` for the `9.x` version?

Reply:
These numbers reference package names to install, and there's no 9.6 package. This is unrelated to our documentation.

This will currently install the latest available version of each major release. I think it is possible to install a specific version by passing, e.g., `--version 11.7` as a parameter to Chocolately, but I think it is probably better to just pull the latest version.

Reply:
My main worry is that pulling `9` means pulling some random version of `9.x` that is not `9.6` which will then lead to strange errors. If you think that cannot happen, I'm fine.

Reply:
I don't think it can happen unless Chocolately does something really weird and suddenly remaps this to an older version of PostgreSQL.

I am open to running a specific version with `--version <ver>` though, but I think it might actually be better to run on the latest release.